packages/browser/package.json

```diff
@@ -72,7 +72,7 @@
     "node-fetch": "^2.6.0",
     "playwright": "^1.31.1",
     "sinon": "^7.3.2",
-    "webpack": "^4.47.0"
```

Comment:
I would leave this one as webpack 4. 

I think this is the version used in the bundler tests in the browser package and is our only real test that we have compatibility with v4